l1-contracts/src/core/Rollup.sol

```diff
@@ -72,4 +75,24 @@ contract Rollup is IRollup {
 
     emit L2BlockProcessed(l2BlockNumber);
   }
+
+  function _constrainGlobals(bytes calldata _l2Block, uint256 _version) internal view {
```

Comment:
why do we pass `_version` as a param if it will always be the constant?

Reply:
Originally I had it in a lib, forgot to update after 😬